src/Forms/FileField.php

```diff
@@ -85,6 +86,43 @@ public function Field($properties = array())
         return parent::Field($properties);
     }
 
+    public function getAttributes()
+    {
+        $attributes = parent::getAttributes();
+
+        $accept = $this->getAcceptFileTypes();
+        if ($accept) {
+            $attributes = array_merge(['accept' => implode(',', $accept)], $attributes);
+        }
+
+        return $attributes;
+    }
+
+    /**
+     * Returns a list of file extensions (and corresponding mime types) that will be accepted
+     *
+     * @return array
+     */
+    protected function getAcceptFileTypes()
+    {
+        $extensions = $this->getValidator()->getAllowedExtensions();
+        if (!$extensions) {
+            return [];
+        }
+
+        $accept = [];
+        $mimeTypes = HTTP::config()->uninherited('MimeTypes');
+        foreach ($extensions as $extension) {
+            $accept[] = ".{$extension}";
+            // Check for corresponding mime type
+            if (isset($mimeTypes[$extension])) {
```

Comment:
Multiple extensions could share the same mime type; You should ensure `$accept` is unique before returning. Maybe associative array?

Reply:
Just below: `return array_unique($accept);` 😉

Reply:
Ah sorry. :D